### PR TITLE
chore: className here is unnecessary

### PR DESCRIPTION
**Pull request by hoho2017**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/table/demo/virtual-list.md](https://github.com/hoho2017/ant-design/blob/master/components/table/demo/virtual-list.md)

### CODE REVIEW

**Comment by hoho2017** [comment]
className is undefined,it's unnecessary!

File: components/table/demo/virtual-list.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ function VirtualTable(props) {
     >
       <Table
         {...props}
-        className={classNames(className, 'virtual-table')}
```
Comment (afc163) [inline components/table/demo/virtual-list.md:106]: We should remove `import classNames from 'classnames';`.

Comment (yoyo837) [reply components/table/demo/virtual-list.md:106]: We can’t, line 86 uses that.

Comment (hoho2017) [reply components/table/demo/virtual-list.md:106]: yes, i also know it

Comment (hoho2017) [reply components/table/demo/virtual-list.md:106]: i think just remove the unnecessary classname

**Pull request merged**